audio/Cargo.toml

```diff
@@ -4,24 +4,28 @@ version = "0.1.0"
 authors = ["Paul Lietar <paul@lietar.net>"]
 description="The audio fetching and processing logic for librespot"
 license="MIT"
+edition = "2018"
 
 [dependencies.librespot-core]
 path = "../core"
 version = "0.1.0"
 
 [dependencies]
+aes-ctr = "0.6"
 bit-set = "0.5"
 byteorder = "1.3"
+bytes = "0.4"
 futures = "0.1"
-lewton = "0.9"
+lewton = "0.10"
+ogg = "0.8"
 log = "0.4"
-num-bigint = "0.2"
+num-bigint = "0.3"
 num-traits = "0.2"
 tempfile = "3.1"
-aes-ctr = "0.3"
+backtrace = "0.3"
 
-librespot-tremor = { version = "0.1.0", optional = true }
-vorbis = { version ="0.1.0", optional = true }
+librespot-tremor = { version = "0.2.0", optional = true }
+vorbis = { version ="0.0.14", optional = true }
```

Comment:
This seems like a step backwards? Why is that?

Reply:
It's the master of librespot that has even moved to 0.0 which I think is 'don't care"

Reply:
I should add the lewton is the one used by default, not vorbis, so unless spotty has a different command line, it should not even matter